home/import_whatsapp_templates.py

```diff
@@ -97,12 +97,15 @@ def create_whatsapp_template_from_row(self, row: "ContentRow") -> None:
             )
         self.seen_slugs.add(page_id)
 
-        if row.category not in WhatsAppTemplate.Category.values:
+        # Convert category to uppercase
+        category = row.category.upper()
```

Comment:
We could also maybe do a `strip` here, then we can also ignore leading and trailing whitespace